aspnetcore/blazor/get-started.md

```diff
@@ -48,7 +48,7 @@ To get started with Blazor, follow the guidance for your choice of tooling:
 1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) preview template by running the following command:
```

Comment:
@danroth27 do we want to still call this as `optional` given the discovery we had regarding how templatesare being discovered by VS?
```suggestion
1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) preview template by running the following command:
```